src/pages/Portfolio/LendingStats/LendingStatsPage.vue

```diff
@@ -70,7 +65,12 @@
 						show-more-id="show-more-fieldpartner"
 						lend-new-id="lend-new-fieldpartner"
 					/>
-					<hr class="tw-border-tertiary tw-my-4">
+					<hr class="tw-border-tertiary tw-mt-4">
+					<badges-section
```

Comment:
Why was the section moved down?

Reply:
A req from Shehaam in the ticket:
`Move badges to the bottom of the page as well.`

Reply:
Ah, I missed the req, thanks!